x/concentrated-liquidity/incentives.go

```diff
@@ -244,6 +245,21 @@ func calcAccruedIncentivesForAccum(ctx sdk.Context, accumUptime time.Duration, l
 		// Incentives to emit per unit of qualifying liquidity = total emitted / liquidityInAccum
 		// Note that we truncate to ensure we do not overdistribute incentives
 		incentivesPerLiquidity := totalEmittedAmount.QuoTruncate(liquidityInAccum)
+
+		// If truncation occurs, we emit events to alert us of the issue.
+		if incentivesPerLiquidity.IsZero() && !totalEmittedAmount.IsZero() {
+			ctx.EventManager().EmitEvent(sdk.NewEvent(
+				types.IncentiveTruncationPlaceholderName,
+				sdk.NewAttribute("pool_id", strconv.FormatUint(poolID, 10)),
+				sdk.NewAttribute("total_liq", liquidityInAccum.String()),
+				sdk.NewAttribute("per_unit_liq", incentivesPerLiquidity.String()),
+				sdk.NewAttribute("total_amt", totalEmittedAmount.String()),
+			))
+
+			telemetry.IncrCounter(1, types.IncentiveTruncationPlaceholderName)
+			ctx.Logger().Error(types.IncentiveTruncationPlaceholderName, "pool_id", poolID, "total_liq", liquidityInAccum, "per_unit_liq", incentivesPerLiquidity, "total_amt", totalEmittedAmount)
```

Comment:
We should create an alert for these and a Grafana dashboard cc: @niccoloraspa @alessandrolomanto